### DISCUSSION TITLE
Basset cache bursting

### DISCUSSION THREAD

**Discussion by GregaUNK** [Laravel-Backpack/community-forum#963] [General] [answered]
Hi,

I see that basset cache bursting relies on fixed hash (substr(md5(base_path('composer.lock')), 0, 12)). 

When I update my css/js assets I want to change that cache bursting string, so users get immediate update. 

How to do that?

**Comment by pxpm** [comment, marked as answer]
Hey @GregaUNK at the moment is not possible. 

The only way to bust the cache is if the `composer.lock` changes.

I've started working on a way to make it configurable in https://github.com/Laravel-Backpack/basset/pull/126

Let's move the conversation over there. You can also follow that PR to see when we add that ability (soon I hope). 

Cheers
